src/server.c

```diff
@@ -52,12 +52,22 @@ int send_response(int fd, char *header, char *content_type, void *body, int cont
 {
     const int max_response_size = 262144;
     char response[max_response_size];
+    time_t rawtime;
 
     // Build HTTP response and store it in response
 
-    ///////////////////
-    // IMPLEMENT ME! //
-    ///////////////////
+    time( &rawtime );
+
+    int response_length = sprintf(response,
+        "%s\n"
+        "Date: %s\n"
+        "Content-Type: %s\n"
+        "%d\n"
+        "Connection: close"
+        "\n"
+        "%s",
+        header, ctime(&rawtime), content_type, content_length, body
```

Comment:
I mentioned this today but I think having it in writing will help reference later: this will handle text files fine, but it won't work for jpgs or pngs. You'll have to figure out another way to copy the body into the response if you want to do that.